buf/registry/module/v1beta1/resource.proto

```diff
@@ -95,7 +95,7 @@ message ResourceRef {
   oneof value {
     option (buf.validate.oneof).required = true;
     // The id of the resource.
-    string id = 1 [(buf.validate.field).string.uuid = true];
```

Comment:
I assume we could write a CEL expression that essentially validates this field is either a UUID or whatever the validation is for a commit id (sounds like it is a UUID without dashes).